src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -2030,32 +2038,37 @@ JNI_METHOD(void, write)
     }
 }
 
+CHIP_ERROR PutPreencodedInvokeRequest(app::CommandSender & commandSender, app::CommandPathParams & path, const ByteSpan & data)
+{
+    ReturnErrorOnFailure(commandSender.PrepareCommand(path, false));
```

Comment:
consider having `, false /* aStartDataStruct */));` to specify what the bool means.

Although to be honest even knowing this is "start data struct" is unclear to me at what it does :(

Reply:
Maybe a comment saying "copycontainer below sets the kFields container already" would help in understanding why this argument is false.

Reply:
done, thanks